css/style.css

```diff
@@ -420,10 +711,129 @@ body {
   }
 
   .history__video_main {
-    width: 681px;
-    height: 374px;
+    width: 696px;
+    height: 390px;
     padding: 0;
   }
+
+  .stories {
+    background: linear-gradient(2.5deg, #fff 0%, #fff 9.9%, #000 10%, #000 100%);
+    color: #fff;
+    height: 939px;
+    margin-top: -10px;
+  }
+
+  .stories__items {
+    justify-content: space-around;
+  }
+
+  .stories__title {
+    margin-top: 0;
+  }
+
+  .story_desktop {
+    display: none;
+  }
+
+  .story {
+    width: 312px;
+  }
+
+  .story__image {
+    height: 166px;
+  }
+
+  .story__title {
+    font-size: 25px;
+    margin-top: 27px;
+  }
+
+  .story__text {
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 30px;
+  }
+
+  .story__button_text {
+    color: #fff;
+    font-size: 20px;
+  }
+
+  .story__date {
+    font-size: 15px;
+    color: #D1D1D1;
```

Comment:
Codacy found an issue: [Expected "#D1D1D1" to be "#d1d1d1" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=6017130)